### ISSUE TITLE
Should not allow to use ngModel as input value when using ReactiveFormsModule

### ISSUE THREAD

**Issue by michaelchiche** [angular/angular#12708]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
When only importing ReactiveFormsModule in our app, we can still use the ngModel as value for our input, even when using a FormControl and setting its value.
And even worse, it overrides the value of the input.

**Expected behavior**
The ngModel input should definitely not override the value of the input if it was already set in the FormControl in the component class.
But I even think that an ngModel input should not be allowed inside an input when only using ReactiveFormsModule

**Minimal reproduction of the problem with instructions**
http://plnkr.co/edit/Qv7coqWlBfLrI2cRmNZt?p=preview

**What is the motivation / use case for changing the behavior?**

Can cause weird behavior, hard to debug things since we have 2 places where we can set the value. And it does not seem like the wanted behavior.

**Please tell us about your environment:**

* **Angular version:** 2.0.X

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by DzmitryShylovich** [comment]
Don't use reactive and model driven form directives on the same form.

**Comment by michaelchiche** [comment]
@DzmitryShylovich can you please read the issue before commenting?

I am saying that I am NOT using the Template driven form but the model (reactive) driven form.
But using ngModel works anyway, and overrides the value of the FormControl

**Comment by DzmitryShylovich** [comment]
http://plnkr.co/edit/lc1M8EAOyM4HL4rkE1zc?p=preview ngModel is not activated

**Comment by michaelchiche** [comment]
Wow, you are not even trying to understand. I am not talking about the ngModel Directive but the ngModel **Input**

**Comment by trotyl** [comment]
Can be closed in favor of #22633?

**Comment by AndrewKushnir** [comment]
Hi @michaelchiche, thanks for reporting the issue. As @trotyl mentioned above, the `ngModel` and `formControl` on the same element was deprecated, so the issue in this ticket seems to be addressed. I will close this ticket, feel free to create a new one of the problem still exists. Thank you.
